### ISSUE TITLE
PageSpeed update - 30 days browser cache expiry time for images is "not enough".

### ISSUE THREAD

**Issue by arunbasillal** [wp-media/wp-rocket#1345]
Example: https://developers.google.com/speed/pagespeed/insights/?url=http%3A%2F%2Farabdietclinic.com%2F

(look for "Serve static assets with an efficient cache policy" under Diagnostics)
https://jmp.sh/8NPR8ph

We should simply do 6 months or even 1 year IMO.

**Comment by arunbasillal** [comment]
We we need a minimum of 4 months. 

3 months gave me this: 

<img width="649" alt="01" src="https://user-images.githubusercontent.com/5751474/48428897-4252b180-e76c-11e8-939a-3e0c439a5b66.PNG">

4 months worked. 

<img width="649" alt="02" src="https://user-images.githubusercontent.com/5751474/48429002-82199900-e76c-11e8-9884-a5d0691b09af.PNG">

I tried to find out the lowest and it was `97 days`. For 96, I got this:

<img width="679" alt="03" src="https://user-images.githubusercontent.com/5751474/48429967-81820200-e76e-11e8-8e2c-7cc3080b35ac.PNG">

In summary. We need at least 97 days. So we can either do 100 days to be nice.
